src/api/app/assets/stylesheets/webui2/monitor.scss

```diff
@@ -0,0 +1,168 @@
+.builderbox {
```

Comment:
How do you know you need all this?

Reply:
Taking into account that we don't plan to change many things, I guess we are going to keep most of this code, maybe with some slight differences.